end2end/fixtures/MyCustomWorkflowFixtures.ts

```diff
@@ -0,0 +1,26 @@
+//const{base}
+
+import{test as baseTest} from "@playwright/test";
+
+type WorkflowFixture = {
+    workflowData:any;
+    
+}
+
+ export const test = baseTest.extend<WorkflowFixture>({

```

Comment:
A fixture is probably unnecessary for this purpose. See [example](https://github.com/department-of-veterans-affairs/LEAF-Automated-Tests/blob/main/end2end/tests/lifecycleSimpleTravel.spec.ts#L8).